protobufs/livekit_ingress.proto

```diff
@@ -45,10 +45,12 @@ message CreateIngressRequest {
   string participant_name = 5;
   // metadata associated with the publishing participant
   string participant_metadata = 10;
-  // whether to pass through the incoming media without transcoding, only compatible with some input types
-  bool bypass_transcoding = 8; 
+  // bool bypass_transcoding = 8 [deprecated = true];
```

Comment:
I assume that it's okay here that we do not consider old `bypass_transcoding` when `bypass_transcoding` is not set.

Reply:
Thinking again about the how we would roll this out, I only deprecated the bypass_transcoding entries instead of removing them. This allows keep using older tools/SDKs after we roll out the server changes.